go.mod

```diff
@@ -65,7 +65,7 @@ require (
 	github.com/prometheus/client_golang v1.13.0
 	github.com/prometheus/client_model v0.2.0
 	github.com/prometheus/common v0.37.0
-	github.com/prometheus/exporter-toolkit v0.7.1
+	github.com/prometheus/exporter-toolkit v0.7.3
```

Comment:
I think this was already bumped to v0.8.2 in #5896 🙂 
Can you rebase this on latest main?

Reply:
Fixes in #5987